test/conformance/kuttl/validate/clusterpolicy/standard/exclude/exclude-namespace/README.md

```diff
@@ -0,0 +1,9 @@
+## Description
+
+This test creates a policy to validate all resources have a `foo: bar` label, excluding the `default` namespace.
```

Comment:
It'd be nice if you added more to this description that this test is specifically about matching on a wildcard but excluding a whole Namespace. The net result should be any Namespaced resource in the excluded Namespace should not be processed.

Reply:
I feel so bad at writing those READMEs correctly 🤣

Reply:
You just don't spend enough time is all...